tokio-console/src/view/task.rs

```diff
@@ -152,17 +247,13 @@ impl TaskView {
 
         overview.push(Line::from(vec![bold("Target: "), Span::raw(task.target())]));
 
-        let title = "Location: ";
-        let location_max_width = stats_area[0].width as usize - 2 - title.len(); // NOTE: -2 for the border
-        let location = if task.location().len() > location_max_width {
-            let ellipsis = styles.if_utf8("\u{2026}", "...");
-            let start = task.location().len() - location_max_width + ellipsis.chars().count();
-            format!("{}{}", ellipsis, &task.location()[start..])
-        } else {
-            task.location().to_string()
-        };
-
-        overview.push(Line::from(vec![bold(title), Span::raw(location)]));
+        let first_line = location_lines_vector[0].clone();
+        location_lines_vector.remove(0);
+        let location_vector = vec![bold(location_heading), Span::raw(first_line)];
+        overview.push(Line::from(location_vector));
+        for line in location_lines_vector {
+            overview.push(Line::from(Span::raw(format!("    {}", line))));
+        }
```

Comment:
Then here we can use the vec slice:

```suggestion
        for line in location_lines_vector[1..] {
            overview.push(Line::from(Span::raw(format!("    {}", line))));
        }
```